sdks/python/apache_beam/dataframe/pandas_doctests_test.py

```diff
@@ -156,19 +157,14 @@ def test_series_tests(self):
             'pandas.core.series.Series.repeat': ['*'],
             'pandas.core.series.Series.replace': ['*'],
             'pandas.core.series.Series.reset_index': ['*'],
-            'pandas.core.series.Series.round': ['*'],
-            'pandas.core.series.Series.searchsorted': ['*'],
-            'pandas.core.series.Series.shift': ['*'],
+            'pandas.core.series.Series.searchsorted': ['*'],  # doctest issue
```

Comment:
What is "doctest issue" referring to? Is it worth filing a JIRA?

Reply:
I haven't dug into this too much, but it seems to be an issue in the underlying doctest parsing code. I've narrowed it down and added a better comment.